operator/apis/config/v1/groupversion_info.go

```diff
@@ -0,0 +1,20 @@
+// Package v1 contains API Schema definitions for the config v1 API group
+//+kubebuilder:object:generate=true
+//+groupName=config.grafana.com
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "config.grafana.com", Version: "v1"}
```

Comment:
- Should this be `config.loki.grafana.com` to avoid conflicts with other "grafana-related" operators?
- Should we already go with `v1` as version?